chaco/plot.py

```diff
@@ -17,34 +17,34 @@
 from .array_data_source import ArrayDataSource
 from .array_plot_data import ArrayPlotData
 from .base_xy_plot import BaseXYPlot
-from .barplot import BarPlot
-from .candle_plot import CandlePlot
-from .colormapped_scatterplot import ColormappedScatterPlot
-from .contour_line_plot import ContourLinePlot
-from .contour_poly_plot import ContourPolyPlot
-from .cmap_image_plot import CMapImagePlot
+from .plots.barplot import BarPlot
```

Comment:
nitpick - can we reorder the imports here? It looks like these imports were originally ordered - and then a few new plot types were added which broke the order - and now we have broken it once more by moving the modules around